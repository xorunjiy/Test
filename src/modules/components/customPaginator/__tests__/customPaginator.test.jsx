import React from 'react';
import CustomPaginator from '../CustomPaginator.jsx';

const props = {
    currentPage: '1',
    pageQuantity: 2,
}

describe('CustomPaginator snapshot', () => {
    it('should correctly render', () => {

        const component = renderer(<CustomPaginator {...props}/>);

        expect(component).toMatchSnapshot();
    });
});
