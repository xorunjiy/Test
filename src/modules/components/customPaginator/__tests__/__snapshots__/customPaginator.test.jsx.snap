// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomPaginator snapshot should correctly render 1`] = `
<div
  className=""
>
  <span
    className=""
  >
    <button
      className=""
      id={1}
    >
      1
    </button>
    <button
      className=""
      id={2}
    >
      2
    </button>
  </span>
</div>
`;
